### PR TITLE
Add missing references to wc_AesInit in the API docs.

### PR DESCRIPTION
**Pull request by anhu**
Fixes ZD 17075

### CODE REVIEW

File: doc/dox_comments/header_files/aes.h [hunk 1]
```diff
@@ -20,6 +20,9 @@
     int ret = 0;
     byte key[] = { some 16, 24 or 32 byte key };
     byte iv[]  = { some 16 byte iv };
+    if (ret = wc_AesInit(gmac.aes, HEAP_HINT, INVALID_DEVID) != 0) {
```
Comment (philljj) [inline doc/dox_comments/header_files/aes.h:23]: Shouldn't this be:
```
wc_AesInit(&enc, ...
```

File: doc/dox_comments/header_files/aes.h [hunk 2]
```diff
@@ -529,6 +545,7 @@ int wc_GmacUpdate(Gmac* gmac, const byte* iv, word32 ivSz,
     Aes enc;
     key[] = { some 16, 24, or 32 byte length key };
 
+    wc_AesInit(&aes, HEAP_HINT, INVALID_DEVID); // Make sure devId updated
```
Comment (philljj) [inline doc/dox_comments/header_files/aes.h:548]: Is the `// Make sure devId updated` needed?

Comment (philljj) [inline doc/dox_comments/header_files/aes.h:548]: This should be
```
wc_AesInit(&enc,
```

**Review by philljj — CHANGES_REQUESTED** [review]

File: doc/dox_comments/header_files/aes.h (hunk 2 above)
Comment (anhu) [reply doc/dox_comments/header_files/aes.h:548]: Yes.  Its in other places. If it belongs there, it belongs here, IMHO.

**Review by philljj — APPROVED** [review]

**Comment by philljj** [comment]
Retest this please

**Pull request merged**